libs/core/src/components/sage-input/sage-input.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, h, Host, Prop } from '@stencil/core';
+
+/**
+ * @slot - Content is placed between the opening closing tags
+ */
+@Component({
+  tag: 'sage-input',
+  styleUrl: 'sage-input.scss',
+  shadow: true,
+})
+export class SageInput {
+  /**
+   * Determines the type of control that will be displayed
+   * @defaultValue "text"
+   */
+  @Prop() type = 'text';
```

Comment:
👏🏾

Reply:
Would be great to show a list of options here or to use inheritedAttributes to allow for all HTML type options.